extensions/oauth/src/test/resources/authorization-code/authorization-code-localcallbackref-config.xml

```diff
@@ -14,22 +14,21 @@
 
     <oauth:token-manager-config name="tokenManagerConfig"/>
 
+    <oauth:authorization-code-grant-type
```

Comment:
@marianogonzalez added the parameter no not allow these elements inline. Still need to confirm with @pablolagreca if the usability for this is fine. If not, we'll have to review the lifecycles of beans nested in dynamic configs again.